bricksrc/system.py

```diff
@@ -0,0 +1,74 @@
+from rdflib import Literal
+from .namespaces import BRICK, TAG, OWL, RDFS
+
+system_subclasses = {
+    "Domestic_Hot_Water_System":{
+        "tags": [TAG.Domestic, TAG.Water, TAG.Hot, TAG.System]
+    },
+    "Electrical_System":{
+        "tags": [TAG.Electrical, TAG.System]
+    },
+    "Gas_System":{
+        "tags": [TAG.Gas, TAG.System]
+    },
+    "Heating_Ventilation_Air_Conditioning_System":{
+        "tags": [TAG.Heat, TAG.Ventilation, TAG.Air, TAG.Conditioning, TAG.System],
+        "subclasses":{
+            "Steam_System":{
+                "tags": [TAG.Steam, TAG.System],
+            },
+            "Water_System": {
+                "tags": [TAG.Water, TAG.System],
+                "subclasses": {
+                    "Chilled_Water_System": {
+                        OWL.equivalentClass: BRICK["CWS"],
+                        "tags": [TAG.Water, TAG.Chilled, TAG.System],
+                    },
+                    "Hot_Water_System": {
+                        OWL.equivalentClass: BRICK["HWS"],
+                        "tags": [TAG.Water, TAG.Hot, TAG.System],
+                        "subclasses":{
+                            "Preheat_Hot_Water_System":{
+                                "tags": [TAG.Hot, TAG.Water, TAG.System, TAG.Preheat, TAG.System]
+                            },
+                            "Reheat_Hot_Water_System":{
+                                "tags": [TAG.Hot, TAG.Water, TAG.System, TAG.Reheat, TAG.System]
+                            },
+                            "Radiation_Hot_Water_System":{
+                                "tags": [TAG.Hot, TAG.Water, TAG.System, TAG.Radiation, TAG.System]
+                            },
+                            "Heat_Recovery_Hot_Water_System":{
+                                "tags": [TAG.Hot, TAG.Water, TAG.System, TAG.Heat, TAG.Recovery, TAG.System]
+                            },
+                        }
+                    },
+                    "CWS": {
+                        OWL.equivalentClass: BRICK["Chilled_Water_System"],
+                        "tags": [TAG.CWS],
+                    },
+                    "HWS": {
+                        OWL.equivalentClass: BRICK["Hot_Water_System"],
+                        "tags": [TAG.HWS]
+                    },
```

Comment:
I feel like we should remove these: now that we're defining out own class structure here, it's less important to provide the abbreviations. When we were modelling the equipment classes before, we wanted to preserve some naming conventions, but I think that's less important for the System stuff (because these are not necessarily explicitly named in the BMS)

Reply:
Do you think we should remove `brick:FCP` as well?

Reply:
Yeah, good idea